templates/slapd.conf.erb

```diff
@@ -76,7 +78,7 @@ access to *
 # Define a config database (cn=config)
 
 database config
-suffix "cn=config"
+#suffix "cn=config"
```

Comment:
@maxdelorme Can you remove the line completely rather than just comment it out? Will merge after that. Thanks.